oat-sa/extension-pcisample
# Fix/INV-707/Missing tooltip

### Related to: https://oat-sa.atlassian.net/browse/INV-707

Requires: 
 - [x] https://github.com/oat-sa/extension-tao-itemqti/pull/2414 (needed for future compilations)

### Summary

Restore the tooltip helper in the textReader PCI.

![image](https://github.com/oat-sa/extension-pcisample/assets/1500098/97fd53c9-3584-498f-b437-e7b925f3ccad)
![image](https://github.com/oat-sa/extension-pcisample/assets/1500098/c6c32a6f-8e29-4ea9-932b-163e489c3975)

Also:
- make sure it works with TAO Advance.
- fix the tooltip not showing up in the item authoring preview (needs the [companion PR](https://github.com/oat-sa/extension-tao-itemqti/pull/2414))

### Details

When migrating the PCI to the IMS standard, we made some surgery to remove or adapt incompatible code. However, it was a tad too aggressive, and [the helper responsible for preparing the tooltip was removed together with the one that brought MathJAX](https://github.com/oat-sa/extension-pcisample/pull/87/files#diff-82c38155dd7370921087076be131fa5e089961d18643a4bdb36253e7ad5d3470L186).

### How to test
- install TAO with the extension pciSamples
- check out the branch
- have an item with the textReader PCI and some tooltips (you can use the ones attached to the ticket)
- check that the tooltip is visible in the preview
- publish a delivery (or pick up an existing one if any)
- take the test and check the tooltip works
